src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  display: block;
+  margin: 0;
+  height: 40px;
+}
+
+.nav {
+  height: inherit;
+  margin: 0;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  height: inherit;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin: 0 20px 0 0;
+  height: 100%;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: black;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  cursor: pointer;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:focus {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
```

Comment:
this pseudo-element shouldn't have border.
Just use background-color and height: 4px instead